book/src/actions.md

```diff
@@ -0,0 +1,221 @@
+# Actions
+
+By now, we have already learned many ways to glue our widgets together.
+We can send messages through channels, emit signals, share reference-counted state and bind properties.
+Now, we will complete our set by learning about actions.
+
+An action is a piece of functionality bound to a certain GObject.
+Let us check out the most simple case where we activate an action without a parameter.
+
+<span class="filename">Filename: listings/actions/1/main.rs</span>
+
+```rust,no_run
+{{#rustdoc_include ../listings/actions/1/main.rs:build_ui}}
+```
+
+First, we created a new [`gio::SimpleAction`](https://gtk-rs.org/gtk-rs-core/stable/latest/docs/gio/struct.SimpleAction.html) which is named "quit" and takes no parameter.
+We also connected a callback which closes the window.
+
+<span class="filename">Filename: listings/actions/1/main.rs</span>
+
+```rust,no_run
+{{#rustdoc_include ../listings/actions/1/main.rs:main}}
+```
+
+One of the most popular reasons to use actions are keyboard accelerators, so we added one here.
+With [`set_accels_for_action`](../docs/gtk4/prelude/trait.GtkApplicationExt.html#tymethod.set_accels_for_action) one can assign one or more accelerators to a certain action.
```

Comment:
When linking to gtk4-rs, local links have the advantage that the git book links to the git docs and the stable book links to the stable docs.
Because of the order "stable" and package-name appears in the URL that unfortunately does not work for gtk-rs-core